vars/common-vars.yml

```diff
@@ -143,6 +143,10 @@ scale_profile: "{{ lookup('env','scale_profile') | default('minimal', true) }}"
 #Set timeout for helm cleanup.
 helm_delete_timeout: 900
 
+# Number of caasp workers to use for the control plane,
+# the remainder will be used as computer hosts
+airship_ucp_control_workers: 2
```

Comment:
Why common-vars? This is airship specific...

Reply:
I put it under the airship-only section of common-vars https://github.com/SUSE-Cloud/socok8s/blob/master/vars/common-vars.yml#L56; we can move that entire section to a new file, but I believe that is a separate task from this PR.